<launch>
    <param name="cloud_topic" type="string" value="/velodyne_points" />
    <param name="odom_topic" type="string" value="/aft_mapped_to_init" />
    
    <include file="$(find aloam_velodyne)/launch/aloam_velodyne_HDL_64.launch"/>

    <node pkg="rviz"  type="rviz"  name="rviz"  args="-d $(find lio_slam)/rviz/mapping.rviz"></node>
    <node pkg="lio_slam"  type="data_pretreat_node"  name="data_pretreat_node"  output="screen"></node>
    <node pkg="lio_slam"  type="back_end_node"  name="back_end_node"  output="screen"></node>
    <node pkg="lio_slam"  type="loop_closing_node"  name="loop_closing_node"  output="screen"></node>
    <node pkg="lio_slam"  type="viewer_node"  name="viewer_node"  output="screen"></node>
</launch>